lib/util/profiles/reviews/custom_rating_bar.dart

```diff
@@ -24,9 +24,9 @@ class CustomRatingBar extends StatelessWidget {
         Icons.star,
         color: Colors.amber,
       ),
+      // ignore: avoid_redundant_argument_values
```

Comment:
Will the autogenerated documentation stay in for the long run or is it just generated to explain the deletion of direction: ?

Reply:
that's not autogenerated, I put that there. Just because I like "itemCount: Review.maxRating". It's more explicit and will not break everything if we decide to change our review system to 3 stars for example.

But I will add a comment to make that more clear